README.md

```diff
@@ -28,7 +28,7 @@ Create a new WebSocket connection using the following.
 - Password is optional.  
 
 ```js
-const obs = new OBSWebSocket('address', 'password');
+const obs = new OBSWebSocket({ address: 'localhost:4444', password: '$up3rSecretP@ssw0rd' });
```

Comment:
Probably, though it likely will be after SGDQ because the waitlist is so long. My hope was to get a CPAP machine before flying out, because AGDQ was _rough_. I'll make due though.